src/Nancy/GlobalizationConfigurationExtensions.cs

```diff
@@ -14,12 +15,14 @@ public static class GlobalizationConfigurationExtensions
         /// <param name="environment">An <see cref="INancyEnvironment"/> that should be configured.</param>
         /// <param name="supportedCultureNames">Cultures that the application can accept</param>
         /// <param name="defaultCulture">Used to set a default culture for the application</param>
+        /// <param name="dateTimeStyles">The <see cref="DateTimeStyles"/> that should be used for date parsing.</param>
         /// <remarks>If defaultCulture not specified the first supported culture is used</remarks>
-        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
+        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null, DateTimeStyles? dateTimeStyles = null)

```

Comment:
This extension should be `Globalization` and not `Cultures` to align it with what it does